.github/workflows/release-doc-build.yml

```diff
@@ -0,0 +1,81 @@
+name: Release Documentation Build
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
```

Comment:
So, this means the workflow runs when a tag starting with *v* is pushed, but no tags including the character string *dev* will be built.

Reply:
Yeah